main.js

```diff
@@ -520,6 +520,17 @@ function windowCreateFinish(json) {
             url: url
         })
     })
+    if (typeof json.windowOptions.appDetails !== "undefined"){
```

Comment:
Can you add the `windows only` logic here?

That would mean adding `process.platform === "win32"` in the `if` condition.